deploy_local_forklift.sh

```diff
@@ -10,7 +10,47 @@ while ! kubectl get deployment -n olm olm-operator; do sleep 10; done
 kubectl wait deployment -n olm olm-operator --for condition=Available=True --timeout=180s
 
 # Deploy operator
-kubectl apply -f forklift-operator/forklift-k8s.yaml
+cat << EOF | kubectl apply -f -
```

Comment:
What's the benefit of inlining this here? It makes us duplicate the same code in deploy_local_forklift_bazel.sh.